packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroupToolbarItem.tsx

```diff
@@ -18,14 +20,20 @@ export const ChipGroupToolbarItem: React.FunctionComponent<ChipGroupToolbarItemP
   ...props
 }: ChipGroupToolbarItemProps) => {
     if (React.Children.count(children)) {
-      return(
-        <ul className={css(styles.chipGroup, styles.modifiers.toolbar)} {...props}>
-          <li>
-            <h4 className={css(styles.chipGroupLabel)}>{categoryName}</h4>
-            <ul className={css(styles.chipGroup)}>{children}</ul>
-          </li>
-        </ul>
+      return (
+        <ChipGroupContext.Consumer>
+            {(HeadingLevel: any) => {
+            return (
+            <ul className={css(styles.chipGroup, styles.modifiers.toolbar)} {...props}>
```

Comment:
I might be getting too nit-picky here but without the linter on I get nervous about code style. You've got 4 spaces of indentation on `{(HeadingLevel: any) => {` and no indentation on `return` or `<ul className=...`. Do you mind fixing it so each has 2 spaces of indendation?

```js
        <ChipGroupContext.Consumer>
          {(HeadingLevel: any) => {
            return (
              <ul className={css(styles.chipGroup)}>{children}</ul>
                ...
              </ul>
            )
          }}
        </ChipGroupContext.Consumer>
```

Sorry if this is overkill.